src/sass/app_2/_common/components/wip.scss

```diff
@@ -0,0 +1,52 @@
+/** @define work-in-progress */
+.work-in-progress {
+    height: 100vh;
+    display: flex;
+
+    @include themify($themes) {
+        background: themed('container_secondary_color');
+    }
+
+    &__content {
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        padding: 0 1.6em;
+    }
+    &__header {
+        @include typeface(--large-center-bold-black);
+        text-transform: none;
+        margin: 0.8em;
+
+        @include themify($themes) {
+            color: themed('wip_color');
+        }
+    }
+    &__text {
+        @include typeface(--title-center-normal-black);
+        text-transform: none;
+        margin: 0 1.6em 2.4em;
+
+        @include themify($themes) {
+            color: themed('wip_color');
+        }
+    }
+    &__btn {
+        height: 42px;
+        cursor: pointer;
+
+        @include themify($themes) {
+            background: themed('wip_color');
+        }
+
+        &--span {
+            font-weight: normal;
```

Comment:
```
text-transform: none;
font-size: 1.6em;
```

Reply:
the text transform follows other buttons design

Reply:
but but
![image](https://user-images.githubusercontent.com/5536948/55367770-3308b080-5521-11e9-8049-7d50d814ca4b.png)

Reply:
we should check it all over app, we have another place with long text which was all caps in designs, I'll start a discussion in slack